Foxtrot/Tests/CRASanitizeTest.cs

```diff
@@ -16,55 +16,44 @@
 using System.Text;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.IO;
 using System.Xml.Linq;
+using Xunit;
+using Xunit.Abstractions;
 
 namespace Tests
 {
-  [TestClass]
   public partial class Test
   {
-    private TestContext testContextInstance;
+    private readonly ITestOutputHelper _testOutputHelper;
 
-    /// <summary>
-    ///Gets or sets the test context which provides
-    ///information about and functionality for the current test run.
-    ///</summary>
-    public TestContext TestContext
+    public Test(ITestOutputHelper testOutputHelper)
     {
-      get
-      {
-        return testContextInstance;
-      }
-      set
-      {
-        testContextInstance = value;
-      }
+        _testOutputHelper = testOutputHelper;
     }
 
-    private static void Verify(string assemblyName, string frameworkPath)
+    private static void Verify(ITestOutputHelper testOutputHelper, string assemblyName, string frameworkPath)
     {
-      var path = Path.Combine(@"..\..\..\Microsoft.Research\Contracts\bin\debug", frameworkPath);
+      var path = Path.Combine(@"..\..\..\..\Microsoft.Research\Contracts\bin\Debug", frameworkPath);
+      string originalPath = Path.Combine(@"..\..\..\..\Microsoft.Research\Imported\ReferenceAssemblies", frameworkPath, assemblyName + ".dll");
 
-      string originalPath = Path.Combine(Path.Combine(@"..\..\..\Microsoft.Research\Imported\ReferenceAssemblies", frameworkPath), assemblyName + ".dll");
-      if (originalPath == null)
-      {
-        Assert.Fail("Can't find original assembly {0}", assemblyName);
-        return;
-      }
-      var checker = new CRASanitizer.Checker(Console.Out);
+      StringWriter stringWriter = new StringWriter();
+      var checker = new CRASanitizer.Checker(stringWriter);
 
       var contractAssemblyName = assemblyName + ".Contracts.dll";
-      Console.WriteLine("Checking {1} {0} for errors...", assemblyName, frameworkPath);
+      testOutputHelper.WriteLine("Checking {1} {0} for errors...", assemblyName, frameworkPath);
 
+      testOutputHelper.WriteLine("  Contract Path: {0}", Path.GetFullPath(Path.Combine(path, contractAssemblyName)));
+      testOutputHelper.WriteLine("  Original Path: {0}", Path.GetFullPath(originalPath));
 
       var errors = checker.CheckSurface(Path.Combine(path, contractAssemblyName), originalPath);
 
-      Assert.AreEqual(0, errors, "Found {0} errors in contract reference assembly {1}", errors, contractAssemblyName);
-      Console.WriteLine("... done.");
+      testOutputHelper.WriteLine(stringWriter.ToString());
+      Assert.Equal(0, errors);
```

Comment:
I would suggest to keep original error message. Otherwise it will take some time to figure out why test failed.